client/keygen.go

```diff
@@ -0,0 +1,20 @@
+package client
+
+import (
+	"github.com/xlab-si/emmy/crypto/common"
+	"github.com/xlab-si/emmy/crypto/dlog"
+	"github.com/xlab-si/emmy/crypto/groups"
+	"math/big"
+)
+
+// GenerateMasterKey generates a master secret key, representing a random integer betweeen
+// 0 and order of the group.
+func GenerateMasterKey(group *groups.SchnorrGroup) *big.Int {
+	return common.GetRandomInt(group.Q)
+}
+
+// GenerateMasterKeyEC generates a master secret key to use with elliptic curve schemes.
+func GenerateMasterKeyEC(curve dlog.Curve) *big.Int {
+	discreteLog := dlog.NewECDLog(curve)
+	return common.GetRandomInt(discreteLog.OrderOfSubgroup)
+}
```

Comment:
Not sure what would be the best place to put these two functions. The master keys will be generated differently for different schemes (where one scheme is pseudonymsys but for example new is coming that is based on RSA group), however at the same time they will probably always simply return a random element  < group.Order (as it is the case in these two functions). So we might simply use group.GetRandomElement. Note that each group should have such a method - Schnorr group has it, while EC group needs to be refactored (to expose the same methods as Schnorr group) and moved into groups package.